src/shared/utils/common.util.ts

```diff
@@ -42,15 +42,24 @@ export function sleep(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
-export function formatAssetDisplayValue(value: any) {
+export function humanizeValue(value: number) {
+  const formatter = new Intl.NumberFormat("en-US", {
```

Comment:
We should try to use the project library for Internationalization and keep consistency. Linguijs can do this through the [.number()](https://lingui.js.org/ref/core.html?highlight=currency#I18n.number) formatter function in the library

Using it will also take into consideration the current locale selected in the app and keep a more consistent UI if that locale dictated different rules on formatting